CMSgov/design-system
# [NO TICKET] stylelint cleanup

## Summary

This is a combination of fixing and ignoring the majority of CSS stylelint errors we've had. I've added PR comments to explain decisions.

I should note I left errors in packages/docs/src/styles/index.scss unaddressed as it might warrant taking a deeper look at it. Errors in this file are:

- `scss/at-import-no-partial-leading-underscore`
- `no-invalid-position-at-import-rule`

## How to test

1. Pull down branch
2. Run `yarn lint:stylelint`

You should only see errors relating to the index.scss file mentioned above.

### If this is a change to code:


### If this is a change to documentation:
